tests-system/lobster-python/rbt-output-file/default-scenario/expected-output/basic.lobster

```diff
@@ -0,0 +1,65 @@
+Written output for 3 items to basic.lobster
+==========
```

Comment:
These two lines must be removed when using `run_tool_tests.py`. Instead, the first line must be moved into `std.out.txt`.